backend/uclapi/uclapi/settings.py

```diff
@@ -198,3 +198,9 @@
     FAIR_USE_POLICY = list(fp)
 
 REDIS_UCLAPI_HOST = os.environ.get("REDIS_UCLAPI_HOST")
+
+# Celery Settings
+CELERY_BROKER_URL = 'redis://' + REDIS_UCLAPI_HOST + ':6379/0'
```

Comment:
We have to have them separate because `StrictRedis` takes just the server as a parameter, so we would have to store the same value twice.